src/files.py

```diff
@@ -28,7 +28,7 @@ def appendFile(fileName,data):
         print("Error reading from file")
     try:
         f2=open(fileName,'w')
-        json.dump(existing_content,f2)
+        json.dump(existing_content,f2, sort_keys=True, indent=4)
```

Comment:
I am not sure why are we writing a list of dict also. Can we just write `existing_content.pop()` or something?
Also why are we appending instead of overwriting?